components/raftstore/src/store/worker/split_config.rs

```diff
@@ -25,9 +24,8 @@ pub struct SplitConfig {
     pub split_contained_score: f64,
     pub detect_times: u64,
     pub sample_num: usize,
-    pub sample_threshold: i32,
-    pub size_threshold: u64,
-    pub key_threshold: u64,
+    pub sample_threshold: u64,
+    pub byte_threshold: usize,
```

Comment:
I think this change breaks backward compatibility.

If a user set the config `size_threshold` in their config files and it does not take effects after they upgrade to the newer version.

Reply:
I created an issue https://github.com/tikv/tikv/issues/9816